index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <title>My portoflio</title>
+</head>
+<body>
+    <nav class="nav_bar">
+        <div class="logo"> <h2>Mylogo</h2></div> 
```

Comment:
- [ ] [OPTIONAL] Great work including Mylogo word here 💯. However, I kindly suggest you personalize it a little bit since this is your personal portfolio web 🤓.